spec/features/authentication_spec.rb

```diff
@@ -1,6 +1,6 @@
 require 'rails_helper'
 
-RSpec.describe 'Authentication', driver: :selenium_chrome, js: true do
+RSpec.describe 'Authentication', driver: :selenium_chrome_headless, js: true do
```

Comment:
I believe there is a way to configure the default driver in the capybara config to not specify it in every test. For example, if you will need to debug your UI tests, you will be able to do it by simply defining an env variable and all the tests will run the browser

Just a sample, probably your config will be different:

```ruby
Capybara.server = :puma, { Silent: true }

Capybara.register_driver :chrome do |app|
  Capybara::Selenium::Driver.new(app, browser: :chrome)
end

Capybara.register_driver :headless_chrome do |app|
  chrome_options = Selenium::WebDriver::Chrome::Options.new(args: ['--headless'])

  Capybara::Selenium::Driver.new(app, browser: :chrome, options: chrome_options)
end

if ENV['FORCE_CHROME']
  Capybara.default_driver     = :chrome
  Capybara.javascript_driver  = :chrome
else
  Capybara.javascript_driver  = :headless_chrome
end
```